### PR TITLE
Bump dev-tools versions

### PR DESCRIPTION
**Pull request by Cjkjvfnby**

### CODE REVIEW

File: default/python/AI/PriorityAI.py [hunk 1]
```diff
@@ -519,15 +519,15 @@ def _calculate_military_priority():
                 status.get("fleetThreat", 0) ** 0.5
                 + 0.8 * status.get("max_neighbor_threat", 0) ** 0.5
                 + 0.2 * status.get("neighborThreat", 0) ** 0.5
-                + monster_threat ** 0.5
+                + monster_threat**0.5
```
Comment (geoffthemedio) [inline default/python/AI/PriorityAI.py:522]: Seems unrelated to the title...?

Comment (Cjkjvfnby) [reply default/python/AI/PriorityAI.py:522]: The new version of code formatter (black) works in a slightly different way.
